scripts/app-stop.sh

```
@@ -0,0 +1,50 @@
+PID_FILE="$SCRIPT_DIR/.app.pid"
+
+cd "$PROJECT_ROOT"
+
+# 1. 앱 종료
+if [ -f "$PID_FILE" ]; then
+    APP_PID=$(cat "$PID_FILE")
+    if kill -0 "$APP_PID" 2>/dev/null; then
+        echo "[INFO] 앱 종료 중 (PID: $APP_PID)..."
+        kill "$APP_PID"
+        # graceful shutdown 대기 (최대 10초)
+        for i in $(seq 1 10); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)